### PR TITLE
Feature - Add Wallet Follow and Unfollow

### PR DESCRIPTION
**Pull request by midgerate**
A new feature to follow and unfollow a particular wallet.

### CODE REVIEW

**Comment by midgerate** [comment]
Draft because its blocked by https://github.com/snapshot-labs/snapshot.js/pull/534   
I need to know the correct snapshot.js version.

File: src/helpers/database/schema.sql [hunk 1]
```diff
@@ -122,6 +122,17 @@ CREATE TABLE follows (
   INDEX created (created)
 );
 
+CREATE TABLE walletfollows (
```
Comment (bonustrack) [inline src/helpers/database/schema.sql:125]: We shouldnt make a new table for it, we should instead use "follows" table, but add a field "type", type can be "space" or "account", later we can also merge "subscription" there.

Comment (bonustrack) [reply src/helpers/database/schema.sql:125]: I'd call it "following"

Comment (bonustrack) [reply src/helpers/database/schema.sql:125]: And add a column "type"

File: src/ingestor/typedData/index.ts [hunk 2]
```diff
@@ -42,14 +42,23 @@ export default async function ingestor(body) {
   let type = hashTypes[hash];
 
   if (
-    !['settings', 'alias'].includes(type) &&
+    !['settings', 'alias', 'walletFollow', 'walletUnfollow'].includes(type) &&
```
Comment (samuveth) [inline src/ingestor/typedData/index.ts:45]: Why does walletFollow and walletUnfollow need to added here?

**Comment by bonustrack** [comment]
Closing this for now as it wont be prioritized anytime soon

**Pull request closed without merging**